diff --git a/src/components/qualityOptions.js b/src/components/qualityOptions.js
index bf59ad166..50b375219 100644
--- a/src/components/qualityOptions.js
+++ b/src/components/qualityOptions.js
@@ -31,27 +31,26 @@ export function getVideoQualityOptions(options) {
     }
 
     // Quality options are indexed by bitrate. If you must duplicate them, make sure each of them are unique (by making the last digit a 1)
-    if (maxAllowedWidth >= 3800 && hostScreenWidth >= 1930) {
-        qualityOptions.push({ name: '4K - 120 Mbps', maxHeight: 2160, bitrate: 120000000 });
-        qualityOptions.push({ name: '4K - 80 Mbps', maxHeight: 2160, bitrate: 80000000 });
+    if (maxAllowedWidth >= 3800) {
+        qualityOptions.push({ name: '4K - 8 Mbps', maxHeight: 2160, bitrate: 8000000 });
+        qualityOptions.push({ name: '4K - 6 Mbps', maxHeight: 2160, bitrate: 6000000 });
+    }
+    if (maxAllowedWidth >= 2500) {
+        qualityOptions.push({ name: '2K - 4 Mbps', maxHeight: 1440, bitrate: 4000000 });
+        qualityOptions.push({ name: '2K - 3.5 Mbps', maxHeight: 1440, bitrate: 3500000 });
     }
     // Some 1080- videos are reported as 1912?
-    if (maxAllowedWidth >= 1900 && hostScreenWidth >= 1290) {
-        qualityOptions.push({ name: '1080p - 60 Mbps', maxHeight: 1080, bitrate: 60000000 });
-        qualityOptions.push({ name: '1080p - 40 Mbps', maxHeight: 1080, bitrate: 40000000 });
-        qualityOptions.push({ name: '1080p - 20 Mbps', maxHeight: 1080, bitrate: 20000000 });
-        qualityOptions.push({ name: '1080p - 15 Mbps', maxHeight: 1080, bitrate: 15000000 });
-        qualityOptions.push({ name: '1080p - 10 Mbps', maxHeight: 1080, bitrate: 10000000 });
+    if (maxAllowedWidth >= 1900) {
+        qualityOptions.push({ name: '1080p - 3 Mbps', maxHeight: 1080, bitrate: 3000000 });
+        qualityOptions.push({ name: '1080p - 2 Mbps', maxHeight: 1080, bitrate: 2000000 });
     }
-    if (maxAllowedWidth >= 1260 && hostScreenWidth >= 650) {
-        qualityOptions.push({ name: '720p - 8 Mbps', maxHeight: 720, bitrate: 8000000 });
-        qualityOptions.push({ name: '720p - 6 Mbps', maxHeight: 720, bitrate: 6000000 });
-        qualityOptions.push({ name: '720p - 4 Mbps', maxHeight: 720, bitrate: 4000000 });
+    if (maxAllowedWidth >= 1260) {
+        qualityOptions.push({ name: '720p - 1.5 Mbps', maxHeight: 720, bitrate: 1500000 });
+        qualityOptions.push({ name: '720p - 1 Mbps', maxHeight: 720, bitrate: 1000000 });
     }
     if (maxAllowedWidth >= 620) {
-        qualityOptions.push({ name: '480p - 3 Mbps', maxHeight: 480, bitrate: 3000000 });
-        qualityOptions.push({ name: '480p - 1.5 Mbps', maxHeight: 480, bitrate: 1500000 });
-        qualityOptions.push({ name: '480p - 720 kbps', maxHeight: 480, bitrate: 720000 });
+        qualityOptions.push({ name: '480p - 720 kbps', maxHeight: 480, bitrate: 750000 });
+        qualityOptions.push({ name: '480p - 420 kbps', maxHeight: 480, bitrate: 420001 });
     }
 
     qualityOptions.push({ name: '360p - 420 kbps', maxHeight: 360, bitrate: 420000 });
